content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -46,7 +46,20 @@ tmpfs, Secrets use a tmpfs, etc.)
 Some popular filesystems that support idmap mounts in Linux 6.3 are: btrfs,
 ext4, xfs, fat, tmpfs, overlayfs.
 
-In addition, support is needed in the 
+In addition, support is needed in the container runtime and in the OCI runtime
+(the low level runtime used by the container runtime).
+
+The OCI runtime requirements are as follows:
```

Comment:
```suggestion
Here is a list of OCI runtimes known to provide support:
```
This is important. If someone adds the right support to another bit of code, this page shouldn't become inaccurate because of work done in another project. It's not fair to other projects to list a requirement based on stale information, and as a CNCF policy we avoid getting into that kind of situation.

If we'd more capacity, we could commit to keeping up to date with the ecosystem; we don't have that capacity, so we write in a way that means we don't need to stay current.